llvm/lib/TargetParser/Triple.cpp

```diff
@@ -115,6 +115,30 @@ StringRef Triple::getArchName(ArchType Kind, SubArchType SubArch) {
     if (SubArch == AArch64SubArch_arm64e)
       return "arm64e";
     break;
+  case Triple::dxil:
+    switch (SubArch) {
+    case Triple::NoSubArch:
```

Comment:
Why default is dxil1.0?

Reply:
> Why default is dxil1.0?

No specific reason other than choose the most basic version. 

Your comment [here](https://github.com/llvm/llvm-project/pull/89823#discussion_r1581541937) appeared to indicate a similar default choice for SPIRV (viz., spirv32 being equivalent to spirv32_v1.0).

I am open to suggestions for alternate default DXIL version choices.